src/components/resultsbox/drug-search.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import SearchIcon from "@mui/icons-material/Search";
+import TextField from "@mui/material/TextField";
+import Box from "@mui/material/Box";
+import Autocomplete from "@mui/material/Autocomplete";
+import { DrugList } from "../druglist";
+import { Button } from "@mui/material";
+
+class DrugSearch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      drugSearchValue: "",
+    };
+  }
+
+  handleDrugSelect = (e, newValue) => {
+    const newDrugSearch = newValue.toLowerCase();
+    if (newDrugSearch) {
+      this.props.updateDrugSearch(newDrugSearch);
+      this.setState({ drugSearchValue: "" });
+    }
+  };
+
+  handleResetSearch = (e) => {
+    this.props.resetSearch();
+    this.setState({ drugSearchValue: "" });
+  };
+
+  handleInputChange = (e, newInputValue) => {
+    this.setState({
+      drugSearchValue: newInputValue,
+    });
+  };
+
+  render() {
+    return (
+      <>
+        <Box sx={{ display: "flex", alignItems: "flex-end" }}>
+          <SearchIcon sx={{ color: "action.active", mr: 1, my: 0.5 }} />
+          <Autocomplete
+            id="search-input"
+            sx={{ width: "200px" }}
+            disableClearable
+            options={DrugList.map(
+              (option) =>
+                option.drugName.charAt(0).toUpperCase() +
+                option.drugName.slice(1)
```

Comment:
Would be nice to give this a name via a variable as it not really is easily understood what you are doing here